src/lib/sijoittelu/sijoittelu-service.ts

```diff
@@ -1,32 +1,40 @@
 import { isNullish } from 'remeda';
-import { configuration } from '../configuration';
 import { client } from '../http-client';
 import { AjastettuSijoittelu } from '../types/sijoittelu-types';
+import { getConfiguration } from '@/lib/configuration/client-configuration';
 
 export async function kaynnistaSijoittelu(hakuOid: string) {
-  await client.post(configuration.kaynnistaSijoittelu({ hakuOid }), {});
+  const configuration = await getConfiguration();
+  await client.post(
+    configuration.routes.sijoittelu.kaynnistaSijoittelu({ hakuOid }),
+    {},
+  );
 }
 
 export async function sijoittelunStatus(
   hakuOid: string,
 ): Promise<{ valmis: boolean; ohitettu: boolean; tekeillaan: boolean }> {
+  const configuration = await getConfiguration();
   const response = await client.get<{
     valmis: boolean;
     ohitettu: boolean;
     tekeillaan: boolean;
-  }>(configuration.sijoittelunStatus({ hakuOid }));
+  }>(configuration.routes.sijoittelu.sijoittelunStatus({ hakuOid }));
   return response.data;
 }
 
 export async function getAjastettuSijoittelu(
   hakuOid: string,
 ): Promise<AjastettuSijoittelu | null> {
+  const configuration = await getConfiguration();
```

Comment:
Miksi tässä on await, vaikka getConfiguration ei ole asynkroninen? Näitä on muuallakin.